AppCenter/AppCenter/MSACNetworkRequestsProtocol.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
You should add this file to "include" folder and reference it in the umbrella header file "AppCenter.h". Otherwise SPM would not compile AppCenter module.